styles.css

```diff
@@ -4,8 +4,35 @@ body {
   margin: 0 auto;
 }
 
+section {
+	border-bottom: 1px solid #f3f3f3;
+}
+
+section.header {
+	height: 500px;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	flex-flow: column wrap;
+}
+
+h1 {
```

Comment:
We should agree a convention on whether to use rems, ems or pixels. Will add to the agenda for the next team meeting.